lib/api/src/lib/shortcut.ts

```diff
@@ -1,7 +1,7 @@
 import { navigator } from 'global';
 
 // The shortcut is our JSON-ifiable representation of a shortcut combination
-type Shortcut = string[];
+import { KeyCollection, Event } from '../modules/shortcuts';
```

Comment:
@Jessica-Koch I'm hoping I did the merges correctly, It seems to all work. I hope this is in line with how you want it.

It's all in typescript now 🎉